CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Supporting of Fedora 34 platform.
```

Comment:
This will make it more clear what we did.
```suggestion
- Build of packages for Fedora 34 platform.
```

Reply:
Renamed.